1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,24 @@
 function readingList(books) {
-  // Write your code here...
+  for (i = 0; i <= books.length; i++) {
+    let liElement = document.createElement("li");
+    let pElement = document.createElement("p");
+    let bookInfo = books[i].title + " by " + books[i].author;
+    pElement.textContent = bookInfo; // textContent only targets the text portion but leave any HTML alone, but if you go after html it will completely ruin HTML that you had before.
+    if (books[i].alreadyRead === true) {
```

Comment:
All good! just bear in mind that instead of adding classes directly to the li element, you can use a ternary operator to conditionally add a class to the li element based on the value of books[i].alreadyRead